src/onnxocr/predict_det.py

```
@@ -30,22 +36,27 @@ def __init__(self, args):
+        # 实例化前处理操作类
         self.preprocess_op = create_operators(pre_process_list)
         # self.postprocess_op = build_post_process(postprocess_params)
         # 实例化后处理操作类
         self.postprocess_op = DBPostProcess(**postprocess_params)
 
         # 初始化模型
-        self.det_onnx_session = self.get_onnx_session(args.det_model_dir, args.use_gpu)
+        self.det_onnx_session = self.get_onnx_session(args.det_model_dir, args.use_gpu, gpu_id = args.gpu_id)
         self.det_input_name = self.get_input_name(self.det_onnx_session)
         self.det_output_name = self.get_output_name(self.det_onnx_session)
+        log.info("Detection model loaded: {}", args.det_model_dir)
```

Ruff (0.15.20)

[error] Too many arguments for `logging` format string

(PLE1205)